examples/svelte/counter/src/components/Hello.html

```diff
@@ -0,0 +1,39 @@
+<div class="header">
```

Comment:
Could you please remove this header? Just to keep all the Counter examples the same..